Convert word-forms-feedstock to v1 feedstock

rxm7706 opened the pull request:
This PR converts word-forms-feedstock to a v1 recipe and switch the conda build tool to rattler-build.
It has been automatically generated with [feedrattler v0.3.14.dev26+g9fab0e2b0](https://github.com/hadim/feedrattler).

Changes:
- [x] 📝 Converted `meta.yaml` to `recipe.yaml`
- [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
- [x] 🔧 Updated `conda-forge.yml` to use `rattler-build` and `pixi` (optional)
- [x] 🔢 Bumped the build number
- [x] 🐍 Applied temporary fixes for `python_min` and `python_version`
- [x] 🔄 Rerender the feedstock with conda-smithy
- [ ] Ensured the license file is being packaged.

devin-ai-integration[bot] committed:
Convert to v1 feedstock and use pixi as conda install tool

devin-ai-integration[bot] committed:
MNT: Re-rendered with conda-build 25.7.0, conda-smithy 3.51.1, and conda-forge-pinning 2025.10.04.03.00.01

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/recipe.yaml**:

 * ℹ️ No valid build backend found for Python recipe for package `word-forms` using `pip`. Python recipes using `pip` need to explicitly specify a build backend in the `host` section. If your recipe has built with only `pip` in the `host` section in the past, you likely should add `setuptools` to the `host` section of your recipe.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/18245722748](https://github.com/conda-forge/conda-forge-webservices/actions/runs/18245722748). Examine the logs at this URL for more detail.</sub>

devin-ai-integration[bot] committed:
Apply recipe.yaml formatting updates

- Add yaml-language-server schema comment
- Quote version string in context section
- Add <4.0 upper bound to python requirement
- Convert pip check script to pip_check: true in python section

devin-ai-integration[bot] committed:
Apply recipe.yaml formatting updates

- Add yaml-language-server schema comment
- Quote version string in context section
- Add <4.0 upper bound to python requirement
- Convert pip check script to pip_check: true in python section

devin-ai-integration[bot] committed:
Add setuptools to host requirements

Fixes build backend error: 'No valid build backend found for Python recipe'
Python recipes using pip need to explicitly specify a build backend.

Co-Authored-By: rxm7706@protonmail.com <rxm7706@protonmail.com>

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

devin-ai-integration[bot] committed:
Update conda-forge.yml with complete bot automerge settings

Add missing bot configuration:
- bot.automerge: true
- bot.inspection: update-all
- bot.check_solvable: true
- bot.run_deps_from_wheel: true

devin-ai-integration[bot] committed:
Apply recipe.yaml formatting updates

- Add yaml-language-server schema comment
- Quote version string in context section
- Add <4.0 upper bound to python requirement
- Convert pip check script to pip_check: true in python section

rxm7706 committed:
Pin inflect==4.1.0

rxm7706 committed:
pip check false
